dv/uvm/icache/dv/ibex_icache_sim_cfg.hjson

```diff
@@ -23,17 +23,19 @@
   // Import additional common sim cfg files.
   import_cfgs: [
       // Project wide common sim cfg file
-      "{proj_root}/vendor/lowrisc_ip/dv/tools/dvsim/common_sim_cfg.hjson"
+      "{proj_root}/vendor/lowrisc_ip/dv/tools/dvsim/common_sim_cfg.hjson",
+      "{proj_root}/vendor/lowrisc_ip/dv/verilator/memutil_dpi_scrambled_opts.hjson"
   ]
 
   build_modes: [
     {
       name: default
-      en_build_modes: ["{tool}_icache_ecc"]
+      en_build_modes: ["{tool}_icache_ecc",
+                       "{tool}_memutil_dpi_scrambled_build_opts"]
     }
     {
       name: vcs_icache_ecc
-      build_opts: ["-gv tb.ICacheECC=1"]
+      build_opts: ["-gv tb.ICacheECC=1 -gv tb.Scramble=1"]
```

Comment:
This doesn't look quite right to me (because we're not reflecting it in the Riviera config below). But should we just leave it disabled for now anyway, until the DV work is done? Looking at the top-level TB, I think you're never passing a scrambling key, which means that turning on scrambling will essentially disable the cache, no?

Reply:
I agree! I disabled it in ICache block level testbench. In others, it was already turned off too.

Reply:
Yep, I was being stupid and thought this was controlling things for the core_uvm tests. Ignore me :-)